tests/country_synonyms.py

```diff
@@ -0,0 +1,1895 @@
+    ('thesyrianarabrepublic', 'Syria'),
+    ('thetogoleserepublic', 'Togo'),
+    ('thetunisianrepublic', 'Tunisia'),
+    ('theunionofthecomoros', 'Comoros'),
+    ('theunitedarabemirates', 'United Arab Emirates'),
+    ('theunitedmexicanstates', 'Mexico'),
+    ('theunitedrepublicoftanzania', 'Tanzania'),
+    ('theunitedstatesofamerica', 'United States'),
+    ('thevirginislands', 'British Virgin Islands'),
+    ('timorleste', 'East Timor'),
+    ('tj', 'Tajikistan'),
+    ('tk', 'Tokelau'),
+    ('tl', 'East Timor'),
+    ('tm', 'Turkmenistan'),
+    ('tn', 'Tunisia'),
+    ('to', 'Tonga'),
+    ('togolese republic', 'Togo'),
+    ('togolese', 'Togo'),
+    ('togoleserepublic', 'Togo'),
+    ('tojikistan', 'Tajikistan'),
+    ('toçikiston', 'Tajikistan'),
+    ('tr', 'Turkey'),
+    ('trinidadandtobago', 'Trinidad and Tobago'),
+    ('tristandacunha', 'Tristan da Cunha'),
+    ('tt', 'Trinidad and Tobago'),
+    ('tunes', 'Tunisia'),
+    ('tunisian republic', 'Tunisia'),
+    ('tunisianrepublic', 'Tunisia'),
+    ('turkiye', 'Turkey'),
+    ('turkmen', 'Turkmenistan'),
+    ('turkmenia', 'Turkmenistan'),
+    ('turkomen', 'Turkmenistan'),
+    ('turksandcaicosislands', 'Turks and Caicos Islands'),
+    ('tv', 'Tuvalu'),
+    ('tw', 'Taiwan'),
+    ('tz', 'Tanzania'),
+    ('táiwān', 'Taiwan'),
+    ('türkiye', 'Turkey'),
+    ('türkmenistan', 'Turkmenistan'),
+    ('tšād', 'Chad'),
+    ('tūns', 'Tunisia'),
+    ('ua', 'Ukraine'),
+    ('uae', 'United Arab Emirates'),
+    ('ug', 'Uganda'),
+    ('ukrayina', 'Ukraine'),
+    ('ukraїna', 'Ukraine'),
+    ('um67', 'Johnston Atoll'),
+    ('um71', 'Midway Islands'),
+    ('um76', 'Navassa Island'),
+    ('um81', 'Baker Island'),
+    ('um84', 'Howland Island'),
+    ('um86', 'Jarvis Island'),
+    ('um89', 'Kingman Reef'),
+    ('um95', 'Palmyra Atoll'),
+    ('umbuso weswatini', 'Eswatini'),
+    ('umbusoweswatini', 'Eswatini'),
+    ('ummalquwain', 'Umm al-Quwain'),
+    ('umzantsi afrika', 'South Africa'),
+    ('umzantsiafrika', 'South Africa'),
+    ('union of soviet socialist republics', 'USSR'),
+    ('union of the comoros', 'Comoros'),
+    ('unionofsovietsocialistrepublics', 'USSR'),
+    ('unionofthecomoros', 'Comoros'),
+    ('unit states', 'United States'),
+    ('unite states', 'United States'),
+    ('united arab republic', 'Egypt'),
+    ('united mexican states', 'Mexico'),
+    ('united republic of tanzania', 'Tanzania'),
+    ('united sat', 'United States'),
+    ('united staes', 'United States'),
+    ('united stated', 'United States'),
+    ('united states america', 'United States'),
+    ('united states of america', 'United States'),
+    ('united stats', 'United States'),
+    ('united sttes', 'United States'),
+    ('unitedarabemirates', 'United Arab Emirates'),
+    ('unitedarabrepublic', 'Egypt'),
+    ('unitedmexicanstates', 'Mexico'),
+    ('unitedrepublicoftanzania', 'Tanzania'),
+    ('unitedsat', 'United States'),
+    ('unitedstaes', 'U
```

Comment:
There was some weird character encoding stuff going on with them that I couldn’t figure out. Do you want to try uncommenting them and see if you can figure it out? I think it’s pretty unlikely we’ll actually get a letter addressed in this way, but it would be nice for completeness…